projects/plugins/jetpack/modules/search/class-jetpack-instant-search.php

```diff
@@ -140,6 +140,35 @@ public function load_assets_with_parameters( $path_prefix, $plugin_base_path ) {
 		wp_set_script_translations( 'jetpack-instant-search', 'jetpack' );
 		$this->load_and_initialize_tracks();
 		$this->inject_javascript_options();
+
+		// It only inline the translations for the script, but does not load it.
+		$this->inject_translation_for_script(
+			plugins_url(
+				$path_prefix . '_inc/build/instant-search/jp-search.chunk-main-payload.min.js',
+				$plugin_base_path
+			)
+		);
+	}
+
+	/**
+	 * Add inline translations for script `$payload_url` before loading `$before_handle` script.
+	 *
+	 * @param string $payload_url - The payload url for which we load the translations.
+	 * @param string $before_handle - Inline the translations before this handle.
+	 */
+	protected function inject_translation_for_script( $payload_url, $before_handle = 'jetpack-instant-search' ) {
+		global $wp_version;
```

Comment:
Is `$wp_version` a reliable cache-buster in our use case? Wouldn't this value remain unchanged across multiple Jetpack releases?

Reply:
Good catch. We don't really need a version here, the `$wp_version` is only to eliminate the warnings. Changed the value  to `false` and disabled the check.